task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +170,8 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    if (Math.pow((point.x - circle.center.x),2) + Math.pow((point.y - circle.center.y), 2) < Math.pow(circle.radius, 2)) return true; 
+    else return false;
```

Comment:
Please, solve this task using only one return
https://javascript.info/logical-operators